lib/data/repositories/todo_repository_impl.dart

```diff
@@ -14,4 +16,25 @@ class TodoRespositoryImpl extends TodoRespository {
 
   @override
   Future<Todos> getTodos() async => await todoRemoteDatasource.getTodos();
+
+  Future<void> deleteTodoById({String id}) async {
+    try {
+      await todoLocalDatasource.delete(id);
+      await todoRemoteDatasource.deleteTodoById(id);
+    } catch (e) {
+      rethrow;
+    }
+  }
+
+  Future<String> update(TodoEntity todoModel) async {
```

Comment:
should we follow same naming convention ?
updateStatusById ?

Reply:
i'm sorry this code is coming from other branch from update feature, if i change this, i'm afraid that will be conflict if others change it too. or should i change it?